test/parallel/test-vm-debug-context-token.js

```diff
@@ -0,0 +1,12 @@
+var common = require('../common');
+var vm = require('vm');
+var assert = require('assert');
+
+var proto = vm.runInDebugContext('DebugCommandProcessor.prototype');
+proto.prop = true;
+assert.equal(proto.prop, true);
+proto = vm.runInDebugContext('DebugCommandProcessor.prototype');
+assert.equal(proto.prop, true);
+proto = vm.runInDebugContext('DebugCommandProcessor.prototype.prop = true;' +
```

Comment:
Should this line be setting to a different value than true? To be sure it actually does something?